packages/tpetra/core/src/Tpetra_ConfigDefs.hpp

```diff
@@ -133,9 +133,9 @@ namespace Tpetra {
 
 #ifdef TPETRA_ENABLE_DEPRECATED_CODE
   TPETRA_DEPRECATED const ProfileType DynamicProfile = ProfileType(StaticProfile+1);
-#define TPETRA_DEFAULT_PROFILE_TYPE DynamicProfile
+#define TPETRA_DEFAULT_PROFILE_TYPE Tpetra::ProfileType(Tpetra::StaticProfile+1)  // DynamicProfile
```

Comment:
Adding 1 to an enum value might generate a warning.

Reply:
Then line 135 will produce a warning as well.  :)
I did not see any warnings from these lines in my builds.